free-1.md

```diff
@@ -57,12 +104,31 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+It seems to me that the split would probably be 50-50.
+It may be that the language is actually very straightforward, and just difficult to implement.
+However, I believe that would not be the case. The language needs to be O(n) effort, meaning that
+adding the info for a new law or ruling should not increase in work as the overall size of the
+database increases. It also needs to be both easy enough to use, not mired in a programmer's
+perspective, and still powerful enough to be useful.
+I think the language would require a lot of work.
```

Comment:
When I think of the language you've described, it's standard english but with some escape character or GUI element where people can put in references -- it doesn't sound like a very complex language